scripts/cybershake/install_cybershake.py

```diff
@@ -74,81 +72,93 @@ def main():
         action="store_true",
         help="Don't use velocity model perturbations. If this is selected any perturbation files will be ignored.",
     )
-
+    parser.add_argument(
+        "--vm_qpqs_files",
+        action="store_true",
+        help="Use generated Qp/Qs files. If this is selected all events/faults must have Qp and Qs files.",
+    )
+    parser.add_argument(
+        "--ignore_vm_qpqs_files",
```

Comment:
You can make these two arguments exclusive.

You could even store it in one variable. A nullable bool. True - use it, False ignore it and None - don't expect it

Reply:
https://riptutorial.com/python/example/6133/setting-mutually-exclusive-arguments-with-argparse